includes/Widgets/Stories.php

```diff
@@ -115,7 +115,7 @@ public function widget( $args, $instance ) {
 		];
 
 		$story_args = [
-			'posts_per_page' => min( absint( $instance['number_of_stories'] ), 20 ),
+			'posts_per_page' => min( absint( $instance['number_of_stories'] ), 20 ), // phpcs:ignore WordPress.WP.PostsPerPage.posts_per_page_posts_per_page
```

Comment:
This is a new false positive. WordPress.WP.PostsPerPage.posts_per_page_posts_per_page

Reply:
No, there is no issue, which is why I ignored. It is not a hard coded value, so the linter can't check it.

Reply:
Hm, should this be reported then? Since previously it didn't trigger false positive for it.